### PR TITLE
add parameter for listen ip

### PR DESCRIPTION
**Pull request by stefan-ahrefs**
#### Pull Request (PR) description
Set the default listen IP of puppetboard to localhost

### CODE REVIEW

File: manifests/apache/vhost.pp [hunk 1]
```diff
@@ -24,6 +24,7 @@
 class puppetboard::apache::vhost (
   String[1] $vhost_name,
   Stdlib::Unixpath $wsgi_alias                 = '/',
+  Stdlib::IP::Address $ip                      = '127.0.0.1',
```
Comment (smortex) [inline manifests/apache/vhost.pp:27]: This breaks backward compatibility.  Maybe make this optional?
```suggestion
  Optional[Stdlib::IP::Address] $ip            = undef,
```

Comment (stefan-ahrefs) [reply manifests/apache/vhost.pp]: replaced as requested with undef, added parameter description, and renamed commit

Comment (smortex) [reply manifests/apache/vhost.pp]: Thanks!

File: manifests/apache/vhost.pp [hunk 2]
```diff
@@ -24,6 +25,7 @@
 class puppetboard::apache::vhost (
   String[1] $vhost_name,
   Stdlib::Unixpath $wsgi_alias                 = '/',
+  Stdlib::IP::Address $ip                      = undef,
```
Comment (smortex) [inline manifests/apache/vhost.pp:28]: Ah! the `Optional[...]` part is missing

Comment (stefan-ahrefs) [reply manifests/apache/vhost.pp]: added Optional

**Review by smortex — APPROVED** [review]
LGTM!

**Pull request merged**